pep/app/pods/components/word-wheel/component.ts

```diff
@@ -34,16 +39,20 @@ export default class WordWheel extends Component<WordWheelArgs> {
         return this.args.apiCore ?? 'docs';
     }
 
+    get searchType() {
+        return this.args.searchType ?? WordWheelSearchType.WORD;
+    }
+
     /**
      * Loads word wheel suggestions for the current word in the input value
      * @param {string} currentWord
      * @returns {Promise<FlTypeaheadSuggestion[]>}
      */
     @action
-    async loadSuggestions(currentWord?: string): Promise<FlTypeaheadSuggestion[]> {
+    async loadSuggestions(currentWord?: string, text?: string): Promise<FlTypeaheadSuggestion[]> {
         if (currentWord?.length) {
             const params = removeEmptyQueryParams({
-                word: currentWord,
+                word: this.searchType === WordWheelSearchType.WORD ? currentWord : text,
```

Comment:
I probably would've done this at the core `fl-typeahead` component level that the word-wheel is built on, as it would allow it to completely skip a lot of the complex logic it is doing to provide and insert that `currentWord` as the user types, but this is fine.